README.md

```diff
@@ -1,31 +1,114 @@
-<!--- [![Continuous Integration](https://github.com/iterative/vscode-dvc/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/iterative/vscode-dvc/actions/workflows/continuous-integration.yml) --->
-<!--- [![Cross-Platform Test](https://github.com/iterative/vscode-dvc/actions/workflows/cross-platform-test.yml/badge.svg)](https://github.com/iterative/vscode-dvc/actions/workflows/cross-platform-test.yml) --->
-<!-- [![DVC CLI Output Test](https://github.com/iterative/vscode-dvc/actions/workflows/dvc-cli-output-test.yml/badge.svg)](https://github.com/iterative/vscode-dvc/actions/workflows/dvc-cli-output-test.yml) -->
+[![Version](https://vsmarketplacebadge.apphb.com/version-short/iterative.dvc.svg)](https://https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+[![Installs](https://vsmarketplacebadge.apphb.com/installs/iterative.dvc.svg)](https://https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+[![Downloads](https://vsmarketplacebadge.apphb.com/downloads/iterative.dvc.svg)](https://https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+[![Ratings](https://vsmarketplacebadge.apphb.com/rating-short/iterative.dvc.svg)](https://https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
 
+[![Continuous Integration](https://github.com/iterative/vscode-dvc/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/iterative/vscode-dvc/actions/workflows/continuous-integration.yml)
 [![Maintainability](https://api.codeclimate.com/v1/badges/fb243c31ea059c0038b2/maintainability)](https://codeclimate.com/repos/608b5886f52398018b00264c/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/fb243c31ea059c0038b2/test_coverage)](https://codeclimate.com/repos/608b5886f52398018b00264c/test_coverage)
 
-A [Visual Studio Code](https://code.visualstudio.com/)
-[extension](https://marketplace.visualstudio.com/VSCode) that aims to allow
-users of all technical backgrounds to effectively use [DVC](https://dvc.org/).
-
-## Current State
-
-We want early adopter feedback! Please let us know what you like and don't like
-about the extension. Feel free to reach out in this repository's
-[issues](https://github.com/iterative/vscode-dvc/issues) or via any of the other
-existing [support channels](https://dvc.org/support).
-
-## Quick Start
-
-- **Step 1.**
-  [Install a supported version of DVC on your system](https://dvc.org/doc/install)
-- **Step 2.**
-  [Install the DVC extension for Visual Studio Code](https://code.visualstudio.com/docs/editor/extension-gallery).
-- **Step 3.** See Walkthrough.
+Run, compare, visualize, and track machine learning experiments right in VS
+Code. This extension uses [DVC](https://dvc.org/), an open-source data
+versioning and ML experiment management tool. No additional services or
+databases are required.
+
+![animation](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/resources/overview.gif)
```

Comment:
So we want both a banner AND this animation this close to one another? The idea of not having to scroll to get the full value proposition can't apply.

TBH I'm not loving the banner as it is now (it's not horizontal): maybe it could float on the right or right side of the title and banners?

Reply:
@maxagin